### ISSUE TITLE
Changes needed for ARIA8: Using aria-label for link purpose 

### ISSUE THREAD

**Issue by spanchang** [w3c/wcag#1985]
Two changes: one for Description and the other for Test Procedure: 
Description includes:
"The aria-label attribute provides a way to place a descriptive text label on an object, such as a link, when there are no elements visible on the page that describe the object. If descriptive elements are visible on the page, the aria-labelledby attribute should be used instead of aria-label. Providing a descriptive text label lets a user distinguish the link from links in the Web page that lead to other destinations and helps the user determine whether to follow the link". 
After stating the above, everything that follows is redundant and even confusing. Therefore I  recommend deleting this text:
"In some assistive technologies the aria-label value will show in the list of links instead of the actual link text.
Per the Accessible Name and Description Computation and the HTML to Platform Accessibility APIs Implementation Guide, the aria-label text will override the text supplied within the link. As such the text supplied will be used instead of the link text by AT. Due to this it is recommended to start the text used in aria-label with the text used within the link. This will allow consistent communication between users."
 
Suggested Tests Procedure
For link elements that use aria-label:
1. The purpose of the link is indicated solely by its context and visual appearance without a text identifier   
2. The value of aria-label attribute properly describes the purpose of the link in the context in which it is presented
Expected Results
Both Items  1 and 2 are  true

Thanks,
Sailesh

**Comment by spanchang** [comment]
The test procedure  should begin with:
"For each object where a aria-label attribute is present".
I accidently omitted it from suggested procedure.
Thanks

**Comment by alastc** [comment]
HI @spanchang,

I realise this is to align with Label in Name, but we generally don't mix criteria / techniques if we can help it. The examples & text don't fail or clash with Label in Name, so we're unlikely to update it on that basis.

I'm also not clear what the change to the procedure is trying to achieve. The 1st step appears to go beyond the goal of the technique? It isn't clear to me what the context-setting is trying to achieve.

**Comment by spanchang** [comment]
Alastair,
The description for ARIA8  clearly states, the technique is applicable "when there are no elements visible on the page that describe the object" ... obviously a  text identifier, right? 
When this is true, some of the remaining description can  be done away with including, "the aria-label text will override the text supplied within the link", no?

The amended test procedure  gives effect to the above.
Thanks,
Sailesh
